Readme.md

```diff
@@ -1,24 +1,20 @@
-# WebMemex/memextension
+# WorldBrain's WebMemex
```

Comment:
ah meant to be temporary. I am also not satisfied with it completely. Want to give the WebMemex some sort of credit for it, didn't know how to do it. 

What do you think about "powered by WebMemex"?